vcd/resource_vcd_ipset_test.go

```diff
@@ -51,8 +51,8 @@ func TestAccVcdIpSet(t *testing.T) {
 					resource.TestCheckResourceAttr("vcd_nsxv_ip_set.test-ipset", "description", "test-ip-set-description"),
 					resource.TestCheckResourceAttr("vcd_nsxv_ip_set.test-ipset", "is_inheritance_allowed", "true"),
 					resource.TestCheckResourceAttr("vcd_nsxv_ip_set.test-ipset", "ip_addresses.#", "2"),
-					resource.TestCheckResourceAttr("vcd_nsxv_ip_set.test-ipset", "ip_addresses.2555711295", "192.168.1.1"),
-					resource.TestCheckResourceAttr("vcd_nsxv_ip_set.test-ipset", "ip_addresses.2329977041", "192.168.2.1"),
+					resource.TestCheckTypeSetElemAttr("vcd_nsxv_ip_set.test-ipset", "ip_addresses.*", "192.168.1.1"),
+					resource.TestCheckTypeSetElemAttr("vcd_nsxv_ip_set.test-ipset", "ip_addresses.*", "192.168.2.1"),
```

Comment:
```
					resource.TestCheckTypeSetElemAttr("vcd_nsxv_ip_set.test-ipset", "ip_addresses.*", "192.168.1.1"),
					resource.TestCheckTypeSetElemAttr("vcd_nsxv_ip_set.test-ipset", "ip_addresses.*", "192.168.2.1"),
```
 Do we here match any element in Set?

Reply:
The above two tests check that it contains these both addresses (note the "ip_addresses.*" notation)

Reply:
ok. So that I guess correctly.